contract/vm_callback.go

```diff
@@ -1325,10 +1325,6 @@ func luaGovernance(L *LState, service C.int, gType C.char, arg *C.char) *C.char
 	senderState := curContract.callState.accState
 	receiverState := scsState.accState
 
-	if senderState.AccountID().String() == "A9zXKkooeGYAZC5ReCcgeg4ddsvMHAy2ivUafXhrnzpj" {
```

Comment:
it is for handle exception about vpr calculation in specific contract. ( AmhNcvE7RR84xoRzYNyATnwZR2JXaC5ut7neu89R13aj1b4eUxKp )

I removed this and replace it in vote.go

Reply:
the code below is just a `ClearAid` but you replaced with add and sub votes...
is that correct?

Reply:
yes. it`s purpose is only for exception handling about calculate votingPowerRank.
i repeatedly test this many times. ( but, I couldn't create unit tests, because it is exception handling )

If there are any unexpected problems, they will be discovered in the sync test.

Reply:
I know it is for an exception case, but why the logic is so different? The original code just calls `ClearAid()`. How that call used to affect VPR and vote result?